styleguide/source/_patterns/03-organisms/footer.twig

```diff
@@ -0,0 +1,32 @@
+<footer class="joe__footer">
+  <div class="container">
+    <div class="joe__layout--two-column">
+      <div class="joe__col-one">
+        {% set wordmark = footer_wordmark %}
+        {% include "@molecules/wordmark.twig" %}
+        {% include "@molecules/footer-nav.twig" %}
+      </div>
+      <div class="joe__col-two joe__footer--right">
+        <div class="joe__footer__block">
+          {% set title_label = footer_title_label %}
```

Comment:
Would it make sense to have a footer_block molecule that can accept, title label, paragraph or link as it's values?  It is a bit disjointed to me to have footer_title_label_2 and then footer_phone as separate things currently. It may feel less to to instead to ```include @molecules/footer_block with footer_block_phone```